core/domain/exp_services.py

```diff
@@ -392,12 +468,19 @@ def apply_change_list(exploration_id, change_list):
             elif change.cmd == exp_domain.CMD_DELETE_STATE:
                 exploration.delete_state(change.state_name)
             elif change.cmd == exp_domain.CMD_EDIT_STATE_PROPERTY:
-                state = exploration.states[change.state_name]
+                state: state_domain.State = exploration.states[
+                    change.state_name]
                 if (change.property_name ==
                         exp_domain.STATE_PROPERTY_PARAM_CHANGES):
+                    # Ruling out the possibility of any other type for mypy
+                    # type checking.
```

Comment:
After yesterday's discussion, We'll add more details for this comment and the one below, right?

Reply:
ditto elsewhere comments require more details on why assert is used

Reply:
comment changed